sky/utils/command_runner.py

```diff
@@ -184,8 +184,9 @@ def _get_command_to_run(
             # cluster by 1 second.
             # sourcing ~/.bashrc is not required for internal executions
             command += [
-                'true && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore'
-                f' && ({cmd})'
+                shlex.quote(
+                    'true && export OMP_NUM_THREADS=1 PYTHONWARNINGS=ignore'
+                    f' && ({cmd})')
```

Comment:
```suggestion
                shlex.quote(
                    'true && export OMP_NUM_THREADS=1 '
                    f'PYTHONWARNINGS=ignore && ({cmd})')
```
and reformat? should be able to compress in 2 lines